engine/class_modules/sc_death_knight.cpp

```diff
@@ -2666,6 +2674,106 @@ struct magus_pet_t : public death_knight_pet_t
   }
 };
 
+struct reanimated_shambler_pet_t : public death_knight_pet_t
+{
+  struct necroblast_t : public pet_action_t<reanimated_shambler_pet_t, melee_attack_t>
+  {
+    necroblast_t( reanimated_shambler_pet_t* player, util::string_view options_str ) :
+      pet_action_t( player, "necroblast", player -> find_spell( 334851 ) )
+    {
+      parse_options( options_str );
+    }
+
+    void execute() override
+    {
+      pet_action_t::execute();
+
+      p()->dismiss();
+    }
+
+    void impact( action_state_t* state ) override
+    {
+      pet_action_t::impact( state );
+      p() -> o() -> trigger_festering_wound( state, 1, p() -> o() -> procs.fw_necroblast );
+    }
+  };
+
+  struct travel_t : public action_t
+  {
+    bool executed;
+
+    travel_t( player_t* player ) : action_t( ACTION_OTHER, "travel", player ), executed( false )
+    {
+      may_miss = false;
+      dual     = true;
+    }
+
+    result_e calculate_result( action_state_t* /* s */ ) const override
+    {
+      return RESULT_HIT;
+    }
+
+    block_result_e calculate_block_result( action_state_t* ) const override
+    {
+      return BLOCK_RESULT_UNBLOCKED;
+    }
+
+    void execute() override
+    {
+      action_t::execute();
+      executed = true;
+    }
+
+    void cancel() override
+    {
+      action_t::cancel();
+      executed = false;
+    }
+
+    timespan_t execute_time() const override
+    {
+      return timespan_t::from_seconds( const_cast<travel_t*>( this )->rng().gauss( 2.9, 0.2 ) );
+    }
```

Comment:
Is this the same delay as gargoyle or just WIP?

Reply:
WIP.  There seems to be a significant amount of variance on how long it takes to impact after spawning.  I'm thinking both the mean and standard deviation need to go up from what gargoyle but need to do some more testing

Reply:
https://docs.google.com/spreadsheets/d/1XJkX7tMaIKpxECNBD9a21PyeDbXmUE4JGxH4y4IkoxU/edit?usp=sharing

Pulled all shambler summon/death events out of a 22 minute log, found the time gap between each summon/death and ran those through wolfram alpha.  Got a mean of 3.449 and sample standard deviation of .2846.  I will round those to 3.5 and 0.28 and update the travel rng in the sim accordingly.

Reply:
https://docs.google.com/spreadsheets/d/1XJkX7tMaIKpxECNBD9a21PyeDbXmUE4JGxH4y4IkoxU/edit?usp=sharing

Took a 22 minute log, filtered it down to summon/cast necroblast events and got the delta between each pair of events and ran the values though wolfram alpha.  Got a mean of 4.597 and sample standard deviation of .3399.  I will update the shambler travel rng accordingly.